File: an2_sem4/MPP/Homework/sapt10/cs/FestivalSellpoint/FestivalSellpoint.Network/DTO/BiletDTO.cs
using FestivalSellpoint.Domain;
using System;

namespace FestivalSellpoint.Network.DTO
{
    [Serializable]
    public class BiletDTO : EntityDTO
    {
        public string NumeCumparator { get; }
        public int NrLocuri { get; }
        public SpectacolDTO Spectacol { get; }

        public BiletDTO(string numeCumparator, int nrLocuri, Spectacol spectacol)
        {
            NumeCumparator = numeCumparator;
            NrLocuri = nrLocuri;
            Spectacol = SpectacolDTO.FromSpectacol(spectacol);
        }
    }
}

File: an2_sem4/MPP/Homework/sapt10/java/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/dto/BiletDTO.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.dto;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Entity;
import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Spectacol;

public class BiletDTO extends EntityDTO {
    private String numeCumparator;
    private int nrLocuri;
    private SpectacolDTO spectacol;

    public BiletDTO(){}

    public BiletDTO(String numeCumparator, int nrLocuri, SpectacolDTO spectacol) {
        this.numeCumparator = numeCumparator;
        this.nrLocuri = nrLocuri;
        this.spectacol = spectacol;
    }

    public String getNumeCumparator() {
        return numeCumparator;
    }

    public int getNrLocuri() {
        return nrLocuri;
    }

    public SpectacolDTO getSpectacol() {
        return spectacol;
    }
}
